src/components/Post.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { User } from './User';
+import { CommentList } from './CommentList';
+
+export const Post = post => (
+  <>
+    <div className="post">
+      <h2>{post.title}</h2>
+      <p>{post.body}</p>
+    </div>
+
+    <div className="user">
+      <User {...post.user} />
+    </div>
+
+    <div>
+      <h2 className="comm">COMMENTS</h2>
+      <div className="comments">
+        {post.comments.map(comment => (
+          <div key={comment.id} className="comment">
+            <CommentList {...comment} />
+          </div>
```

Comment:
create CommentList component with list in it

```suggestion
        {post.comments.map(comment => (
          <div key={comment.id} className="comment">
            <Comment {...comment} />
          </div>
```